operations/node_management/Add_Remove_Replace_NCNs/Allocate_NCN_IP_Addresses.md

```diff
@@ -11,7 +11,7 @@ This procedure will perform and verify the following:
 
 ## Procedure
 
-1. Retrieve an API token.
+1. (`ncn-m#`) Retrieve an API token.
```

Comment:
Does this procedure need to be run on a master NCN? Or would it work on any NCN? Should a prerequisites section be added stating that the CLI is configured, like on the other pages? I think this procedure uses CLI commands. If it also uses scripts from the docs repo, then a prerequisite should be added for that too.

Reply:
I definitely wouldn't specify m001 for stuff that could be on any master node. I don't know what our guidance is to customers in terms of where they should do this kind of thing. Anyway, since it's not really related to your PR, leaving it as master is fine for now.